lang/en/auth.php

```diff
@@ -43,7 +43,7 @@
     'email-verification-link' => 'An email verification link has been emailed to you!',
     'enter-recovery'          => 'Please enter one of the recovery codes provided to you when first enabling two-factor authentication.',
     'enter-totp'              => 'Please enter the 6-digit authentication code generated by your app.',
-    'failed'                  => 'These credentials do not match our records.',
```

Comment:
We use this, it can't be deleted:

https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/c64275f0b5dcb3c4c845d5204871adfe24f359d6/app/Providers/FortifyServiceProvider.php#L181